src/lib/raft/raft.c

```diff
@@ -158,6 +159,32 @@ raft_add_vote(struct raft *raft, int src, int dst)
 	return true;
 }
 
+/**
+ * Vote can be revoked if it was volatile (not yet in WAL) and thus wasn't shown
+ * to any other instance yet.
+ */
+static void
+raft_del_vote(struct raft *raft, int src, int dst)
```

Comment:
Maybe `raft_revoke_vote` or `raft_vote_revoke`?

Reply:
I used `raft_voke_vote`. It is consistent with other `raft_<verb>vote` functions like `raft_add_vote`, `raft_sm_schedule_new_vote`, `raft_sm_try_new_vote`.